config/Database_params.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+    class DatabaseParams{
+        protected $db_params = array(
+            'host' => 'localhost',
+            'db_name' => 'chessapidatabase',
+            'username' => 'root',
+            'password' => ''
```

Comment:
This should not be here! Config variables allowed only!
Never commit passwords (except for very very limited cases)!